test/test_chapeldomain.py

```diff
@@ -416,6 +417,7 @@ def test_get_attr_like_prefix(self):
             ('config const n', 'config const '),
             ('blah blah blah blah blah', 'blah blah blah blah '),
             ('enum Color', 'enum '),
+            ('enum constant USA', 'enum constant '),
```

Comment:
Hmm.  Not sure how I feel about this, since `enum constant` isn't Chapel syntax.  It is descriptive, though . . .

Reply:
My understanding is that all of this is the stuff that gets output to the right of any `.. <whatever>::`, e.g. `.. enumconstant:: enum constant USA`, is that correct?  If so, I'd be inclined to not output that second part and just rely on the test on line 414, (so we'd generate `.. enumconstant:: USA` instead, I think?)

Or is this what is used to create those `.. enumconstant:: USA` pieces in the first place?  In which case, it's fine to leave it as is.

(Sorry for not knowing the answer myself)

Reply:
Your understanding is correct, "enum constant" isn't necessary for the `..enumconstant::` to be outputted. My line of thinking for outputting `enumconstant:: enum constant USA` is only because of the cohesiveness with the rest of the documentation, simply a design decision that can be changed :)

Reply:
Then let's bring that up to the rest of the team, or at least Daniel and Ahmad.  I don't want to accidentally steer us in a direction the rest of the team doesn't like just because I'm loud :)